Mutuba/Ride-My-Way3-
# #158834383. Have all API V2 Endpoints working.

### What does the PR do?

Implements the following endpoints:
- POST/api/v2/auth/register
- POST/api/v2/auth/login
- POST/api/v2/rides
- GET/api/v2/rides
- GET/api/v2/rides/ride_id
- PUT/api/v2/rides/ride_id
- DELETE/api/v2/rides/ride_id
- POST/api/v2/rides/ride_id/requests
- GET/api/v2/rides/request_id
- DELETE/api/v2/rides/request_id
- PUT/api/v1/rides/request_id

## How to manually test
- Clone the repo from github. Change directory into Ride-My-Way3- and run python run.py , to start the server.
- Open postman and test all the above endpoints to see the results.
- Remember after registering, login gives you an access token to authenticate you for resources that need token. In Post Man: enter access-token then provide the token generated from login endpoint.

## Relevant pivotal tracker stories
- #158834383
